skyportal/tests/frontend/test_shifts.py

```diff
@@ -223,14 +231,13 @@ def test_shift(
         timeout=30,
     ).click()
 
-    shift_on_calendar = f'//*/span/strong[contains(.,"{form_name}")]'
-    # check for API shift
-    driver.wait_for_xpath(
-        shift_on_calendar,
-        timeout=30,
-    )
+    # scroll to the top of the page
+    driver.execute_script("window.scrollTo(0, 0);")
+    time.sleep(1)
```

Comment:
Why does sleep help here?

Reply:
Let me try to remove it, I had issues with that part of the test but I made other chances that might have fixed it and maybe I can remove this one.